### PR TITLE
fix positioning of VR buttons/labels in WebVR so not obscured by "View Source" when iframed from Examples (fixes #8331)

### LINKED ISSUE #8331: VR buttons obscured by "View Source" button in WebVR Cubes example
http://threejs.org/examples/#webvr_cubes

screenshot 2016-03-09 22 18 40
screenshot 2016-03-09 22 18 59

### PR DESCRIPTION
**Pull request by cvan**
### before

screenshot 2016-03-09 22 18 40
screenshot 2016-03-09 22 18 59
### after

screenshot 2016-03-09 22 26 48
screenshot 2016-03-09 22 29 33

### CODE REVIEW

File: examples/webvr_cubes.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 			.menu {
 				position: fixed;
 				bottom: 20px;
-				right: 20px;
+				left: 20px;
```
Comment (cvan) [inline examples/webvr_cubes.html]: I initially had conditional styles that would put the menu on the left if iframed (`window.top !== window.self`), but I figured it made sense to just keep things in the same position regardless.

should probably use consistent menus for all WebVR examples, but that's not super serious.

File: examples/webvr_cubes.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
 			}
 
 			.button.enabled {
```
Comment (cvan) [inline examples/webvr_cubes.html]: I guess this class was never getting used so I [handled it below](https://github.com/mrdoob/three.js/pull/8332/files#diff-b860e4efd3f299bea60d5ea2f72c89d8R137)

File: examples/webvr_cubes.html [hunk 3]
```diff
@@ -26,7 +26,7 @@
...
-				background-color: rgb(18, 36, 70);
+				background-color: rgb(18, 140, 36);
```
Comment (cvan) [inline examples/webvr_cubes.html]: this turns the button into a nice green

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**